feat(l10n): update translations from Weblate

sudden6 opened the pull request:
@Diadlo please lgtm if you're okay with changing the weblate merge process to squash and rebase, then I'll create a script to automate updating weblate

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qtox/qtox/4981)
<!-- Reviewable:end -->

Diadlo commented:
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

---

Reviewed 18 of 18 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/4981#-:-L6eod4ThE_9RYokzaEQ:b-jbsf8z)*
<!-- Sent from Reviewable.io -->

sudden6 committed:
chore(l10n): update translations from Weblate

Allan Nordhøy (1):
      feat(l10n): update Norwegian Bokmål translation from Weblate

Andreas Kleinert (5):
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate

H Zeng (6):
      feat(l10n): update Chinese (Simplified) translation from Weblate
      feat(l10n): update Chinese (Simplified) translation from Weblate
      feat(l10n): update Chinese (Simplified) translation from Weblate
      feat(l10n): update Chinese (Simplified) translation from Weblate
      feat(l10n): update Chinese (Simplified) translation from Weblate
      feat(l10n): update Chinese (Simplified) translation from Weblate

Johnny (2):
      feat(l10n): update Portuguese translation from Weblate
      feat(l10n): update French translation from Weblate

Jonatan Nyberg (1):
      feat(l10n): update Swedish translation from Weblate

Kaur Männamaa (2):
      feat(l10n): update Estonian translation from Weblate
      feat(l10n): update Estonian translation from Weblate

Kristjan Räts (1):
      feat(l10n): update Estonian translation from Weblate

Moo (1):
      feat(l10n): update Lithuanian translation from Weblate

Nathan Follens (1):
      feat(l10n): update Dutch translation from Weblate

Nikolay Korotkiy (2):
      feat(l10n): update Esperanto translation from Weblate
      feat(l10n): update Finnish translation from Weblate

Slobodan Terzić (1):
      feat(l10n): update Serbian translation from Weblate

Swirti Liberto (1):
      feat(l10n): update French translation from Weblate

Tiger tiger (3):
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate

Uranna 12 (5):
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate

Viktar Vauchkevich (1):
      feat(l10n): update Belarusian translation from Weblate

anonymous (2):
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate

gdqyx3whzs (3):
      feat(l10n): update Macedonian translation from Weblate
      feat(l10n): update Macedonian translation from Weblate
      feat(l10n): update Macedonian translation from Weblate

gorbiWTF (1):
      feat(l10n): update German translation from Weblate

kgkj kgkj (2):
      feat(l10n): update Greek translation from Weblate
      feat(l10n): update Greek translation from Weblate

ms-pc (2):
      feat(l10n): update Spanish translation from Weblate
      feat(l10n): update Spanish translation from Weblate

prhtnsm (3):
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate
      feat(l10n): update German translation from Weblate

remussatala (2):
      feat(l10n): update Romanian translation from Weblate
      feat(l10n): update Romanian translation from Weblate

vasily-mett (1):
      feat(l10n): update Russian translation from Weblate

wang (4):
      feat(l10n): update Chinese (Simplified) translation from Weblate
      feat(l10n): update Chinese (Simplified) translation from Weblate
      feat(l10n): update Chinese (Simplified) translation from Weblate
      feat(l10n): update Chinese (Simplified) translation from Weblate
